### PR TITLE
Jupyter README: update to mention personal account and private workspace

### PR DESCRIPTION
**Pull request by tlvu**
Fixes https://github.com/Ouranosinc/pavics-sdi/issues/151.

Should have been part of PR
https://github.com/bird-house/birdhouse-deploy/pull/16#issuecomment-580799729
(commit
https://github.com/bird-house/birdhouse-deploy/commit/53576cc9d36642c50e4a649ca58fc8339559fd4a).

Dropped the part about " install the `birdy` client locally along with
software to read netCDF files".

### CODE REVIEW

**Comment by tlogan2000** [comment]
Do we possibly want to rename 'public' user to 'demo' .... maybe indicate that the demo instance runs with very limited resources, could be interrupted without notice, etc etc

**Comment by tlvu** [comment]
> Do we possibly want to rename 'public' user to 'demo' .... maybe indicate that the demo instance runs with very limited resources, could be interrupted without notice, etc etc

Sure, feel free to update the wording to your liking.  I'll handle the "technical" part to rename the user once we merge this.

File: docs/source/notebooks/README.ipynb [hunk 1]
```diff
@@ -15,7 +15,9 @@
...
     "\n",
-    "This JupyterLab is not meant as a production environment. It is a public server with no private directory. Your files are public, can be seen by the rest of the world and can be reset at anytime. Use it to explore the system's capability and see whether it may be useful to your work. If it is, install the `birdy` client locally along with software to read netCDF files and you're set to go. \n",
+    "The `public` demo account on this JupyterLab is not meant as a production environment. It is a public account with no private directory and limited computing resources. Your files are public, can be seen by the rest of the world and can be reset at anytime. Use it to explore the system's capability and see whether it may be useful to your work.\n",
+    "\n",
+    "You can request personal account by contacting the email on the login page.  With a personal account, you will have a production environment with private workspace and no computing resource limitation.\n",
```
Comment (huard) [inline docs/source/notebooks/README.ipynb]: Once logged, you need to log out to see the login page again. I suggest we repeat the email here. Also, do we give an account to everyone, do we ask for some kind of description of their purpose ? I'm not suggesting we do, just asking.

**Comment by tlogan2000** [comment]
@tlvu go ahead and merge as is.  We can look to change the name of the user later this week

File: docs/source/notebooks/README.ipynb (hunk 1 above)
Comment (tlvu) [reply docs/source/notebooks/README.ipynb]: >  I suggest we repeat the email here

The email on the login page is customizable through `env.local` file.  Repeating the email here means all the other organizations deploying PAVICS will contact us :(

**Pull request merged**